src/angular.site.json

```diff
@@ -0,0 +1,44 @@
+{
+  "projects": {
+    "microsoft.iis.web-manager": {
+      "architect": {
+        "build": {
+          "configurations": {
+            "wac": {
+              "fileReplacements": [
+                {
+                  "replace": "app/environments/environment.ts",
+                  "with": "app/environments/environment.prod.ts"

```

Comment:
There seems to be one garbage file named environemnt.prod.ts. Please remove the file.

```
C:\gitroot\IIS.WebManager\src>dir app\environments
...
03/28/2019  09:51 AM                78 environemnt.prod.ts
03/28/2019  10:13 AM               543 environment.prod.ts
...

```

Reply:
Fixed